src/pages/home.tsx

```diff
@@ -1,9 +1,108 @@
-import React from "react";
+import './home.css';
+import React from 'react';
+import { ReactComponent as PageLinksIcon } from '../assets/arrow.svg';
+import { ReactComponent as CreateHackathonIcon } from '../assets/createHackathonIcon.svg';
+import { ReactComponent as HackathonAnalyticsIcon } from '../assets/hackathonAnalyticsIcon.svg';
+import { Link } from 'react-router-dom';
 
 export default function Home() {
-  return (
-    <div>
-      <h1>HOME</h1>
-    </div>
-  );
+	// TODO: This user name should be generic.
+	let userName: string = 'John';
+
+	return (
+		<div id="page">
+			<div id="page-header">
```

Comment:
This header could use some additional space above and below to match the specs on the Figma design, it looks very cramped to me right up against the top navigation bar.